<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>625</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>LinkGame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 140, 120);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="timeBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>2</y>
      <width>600</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	border: 1px solid #d3d3d3;
    border-radius: 5px;
    text-align: center;
}
QProgressBar::chunk {
	background-color: #4552a6;
	border-radius: 4px;
}
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="giveTip">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomMap">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);</string>
    </property>
    <property name="text">
     <string>重排</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>giveTip</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
